tests/e2e/workloads/app/test_jenkins_simulation.py

```diff
@@ -38,8 +39,35 @@ class TestJenkinsSimulation(ManageTest):
     """
 
     @workloads
+    @polarion_id("OCS-4668")
+    @bugzilla("2096395")
     def test_git_clone(self, pod):
         """
         git clones a large repository
+        Added test coverage for BZ #2096395
         """
+        csi_cephfsplugin_pod_objs = res_pod.get_all_pods(
+            namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+            selector=["csi-cephfsplugin"],
+        )
+
+        relevant_pod_logs = None
+        func_calls = ["NodeStageVolume", "NodeGetVolumeStats"]
+        error_msg = "System has not been booted with systemd"
+        for pod_obj in csi_cephfsplugin_pod_objs:
```

Comment:
Instead of checking all pods , why we are not checking on node on which pod where cephpvc is mounted and csi-cephfsplugin pod is running. As per https://bugzilla.redhat.com/show_bug.cgi?id=2096395#c32

Reply:
Done